test/justrun.lua

```diff
@@ -72,6 +72,7 @@ function justrun.tarantool(dir, env, args, opts)
     local opts = opts or {}
     assert(type(opts) == 'table')
 
+    env['INPUTRC'] = '/dev/null'
```

Comment:
Nit: I would move it into its own code section (with an empty line before and after) and leave some explanatory comment.

Reply:
Done.